target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -635,7 +644,7 @@ define Device/HG556a-A
   DEVICE_DTS := hg556a-a
   CFE_BOARD_ID := HW556
   CFE_CHIP_ID := 6358
-  CFE_EXTRAS := --rsa-signature "EchoLife_HG556a" --image-offset 0x20000 --block-size 0x10000 --tag-version 8
+  CFE_EXTRAS += --rsa-signature "EchoLife_HG556a" --image-offset 0x20000 --tag-version 8
```

Comment:
Don't you need to override IMAGE_SIZE for hg556 due to the calibration data limiting the usable flash space further?